test/parallel/test-debug-port-from-cmdline.js

```diff
@@ -4,22 +4,21 @@ var spawn = require('child_process').spawn;
 
 var debugPort = common.PORT;
 var args = ['--debug-port=' + debugPort];
```

Comment:
No, there are plenty of parallel tests that listen on a port.  The logic for that is in tools/test.py and test/common.js: the former sets TEST_THREAD_ID which the latter turns into a per-thread port range.

Aside: 'thread' is a bit of a misnomer because they are really separate processes.